vue-frontend/src/components/jobs/WhyCanopas.vue

```diff
@@ -91,13 +93,20 @@ export default {
     };
   },
   mounted() {
-    window.addEventListener("scroll", this.handleScroll);
+    window.addEventListener("scroll", this.handleScroll, this.sendEvent);
```

Comment:
Is this working?
Also we don't need it as you have already added refs and same code in index page